packages/eslint-plugin/tests/rules/prefer-find.test.ts

```diff
@@ -584,5 +590,72 @@ arr.find(f, thisArg);
         },
       ],
     },
+    {
+      code: `
+declare const f: any, g: any;
```

Comment:
[Testing] Nit: here and below, if there's no need to use an `any`, might as well avoid it:

```suggestion
declare const f: unknown, g: unknown;
```

Reply:
it would have a type error without. I was just too lazy to write out the actual type of the filter callback (and the rule doesn't care about the type of the filter callback) so i went for the easiest way to not error :)

Reply:
Ha, gotcha. #8298 -> #8351 tracks making tests error on type unhappiness. cc @auvred - do you think we should start asking folks to try to not have type errors?

Reply:
Personally, I'm +1 to avoid semantic errors, at least in newly added tests :)

Especially when it's not that hard:
```diff
- declare const f: any, g: any;
+ declare const f: (x: unknown) => boolean, g: (x: unknown) => boolean;
```

Reply:
Fair enough, changed